service/controller/resource/scrapeconfigs/resource.go

```diff
@@ -108,18 +109,22 @@ func getTemplateData(cluster metav1.Object, provider string, clients k8sclient.I
 		etcd = fmt.Sprintf("%s:%d", v.Spec.Cluster.Etcd.Domain, v.Spec.Cluster.Etcd.Port)
 	case *v1alpha1.KVMConfig:
 		etcd = fmt.Sprintf("%s:%d", v.Spec.Cluster.Etcd.Domain, v.Spec.Cluster.Etcd.Port)
+	case *corev1.Service:
+		// TODO: find a way to compute etcd url.
```

Comment:
It seems we can use relabelling for the management cluster https://github.com/giantswarm/g8s-prometheus/blob/e927fe8e3f17354b6ee884badd39f15cf0d9840e/helm/g8s-prometheus/templates/prometheus/prometheus-configmap.yaml#L231

Reply:
let's do this in next PRs, when we tackle etcd target.

Reply:
Sure :)